action.yml

```diff
@@ -79,47 +79,7 @@ runs:
     - name: Run tests
       shell: bash
       env:
-        SENTRY_DSN: https://7b09f40b225179cd5d349ff37f409465@o1.ingest.sentry.io/4505784431476736
+        PYTEST_SENTRY_DSN: https://5a620019b5124cbba230a9e62db9b825@o1.ingest.us.sentry.io/6627632
       run: |
-        set +e
         cd self-hosted
         pytest --reruns 1 _integration-test/ --customizations=disabled
-        if [[ $test_return -ne 0 ]]; then
-          echo "Test failed.";
-          sentry-cli send-event -m "Self-hosted e2e action failed in ${{ inputs.project_name }}" --logfile "logs.txt";
```

Comment:
we don't need to send event when this test action fails, alerting can be set up a different way since in s4s we have something that can track CI failures across the getsentry org

Reply:
How are we going to do alerting instead?

Reply:
We will add the workflow file to the alert https://sentry.sentry.io/alerts/rules/details/231322/?alert=104858&referrer=metric_alert_slack&notification_uuid=7d2b426f-4093-4d7c-b298-bc328c582466

Reply:
created new alert rule:
https://sentry.sentry.io/alerts/rules/details/240561/